packages/main/src/Card.hbs

```diff
@@ -27,11 +26,12 @@
 					<div id="{{_id}}-subtitle" class="ui5-card-subtitle ">{{subtitle}}</div>
 				{{/if}}
 			</div>
-			<span part="status" class="ui5-card-status">{{status}}</span>
-		</header>
+			<span id="{{_id}}-status" part="status" class="ui5-card-status">{{status}}</span>
```

Comment:
Noticed that the "status" span has an ID now, should we reference it in aria-labelledby as well,
I could not see where the ID is used and I was just wondering if using it in aria-labelledby was the intention. Sorry if I missed its usage.

Reply:
Yes we should used it, that's why added it and during merge, i guess i have removed it, my mistake.